hveto/tests/test_utils.py

```diff
@@ -43,3 +45,29 @@ def test_write_lal_cache(tmpdir):
 ])
 def test_channel_groups(n, out):
     assert list(utils.channel_groups([1, 2, 3, 4, 5], n)) == out
+
+
+mock_data = EventTable(data=[[1257293167.0], [35.7], [46.2],
```

Comment:
I'd say make this a global variable (defined at the top of file, just after import statements, named with all caps) and format it a little differently with a more descriptive name, something like:

```python
HVETO_RESULTS = EventTable(
    data=[[1257293167.0], [35.7], [46.2], ['H1:GDS-CALIB_STRAIN'],
          ['H1:LSC-POP_A_LF_OUT_DQ'], [152.1195], [1]],
    names=('time', 'peak_frequency', 'snr', 'channel',
           'winner', 'significance', 'round'),
)
```